commands/anime/waifu.js

```diff
@@ -73,12 +73,11 @@ module.exports = class WaifuCommand extends Command {
         } else if (somethingThere) {
             const waifuNumber = args.number;
             var waifu = waifus[waifuNumber];
-            const embed = new Discord.MessageEmbed()
-                .setAuthor(waifu.name, waifu.image)
-                .setDescription(waifu.origin)
-                .setImage(waifu.image)
-                .setFooter(`Waifu Number ${waifuNumber}`)
-                .setColor('#FAC193');
+            embed.setAuthor(waifu.name, waifu.image);
```

Comment:
@Snoowastaken Does this work, with over-writing the const with new values?

Reply:
A single const .MessageEmbed() is being instantiated "embed" and all the embed.blahblah does is set specific arguments for it. So before the embed.setAuthor and so on are actually used, it's just a blank embed with nothing associated with it. For that specific embed.setAuthor() what it is doing is taking the name of the waifu associated with the number the user gives from the waifus.json file and sets the text for the title as the name of the waifu and then it takes the picture of that waifu and sets it as the embed's icon pic.